pmd-java/src/test/java/net/sourceforge/pmd/lang/java/ast/Java16TreeDumpTest.java

```diff
@@ -11,22 +11,18 @@
 
 import net.sourceforge.pmd.lang.ast.ParseException;
 import net.sourceforge.pmd.lang.ast.test.BaseParsingHelper;
-import net.sourceforge.pmd.lang.ast.test.BaseTreeDumpTest;
-import net.sourceforge.pmd.lang.ast.test.RelevantAttributePrinter;
+import net.sourceforge.pmd.lang.java.BaseJavaTreeDumpTest;
 import net.sourceforge.pmd.lang.java.JavaParsingHelper;
 import net.sourceforge.pmd.lang.java.symbols.JElementSymbol;
 import net.sourceforge.pmd.lang.java.types.JPrimitiveType;
 
-public class Java16TreeDumpTest extends BaseTreeDumpTest {
+public class Java16TreeDumpTest extends BaseJavaTreeDumpTest {
     private final JavaParsingHelper java16 =
             JavaParsingHelper.WITH_PROCESSING.withDefaultVersion("16")
                     .withResourceContext(Java15TreeDumpTest.class, "jdkversiontests/java16/");
     private final JavaParsingHelper java16p = java16.withDefaultVersion("16-preview");
     private final JavaParsingHelper java15 = java16.withDefaultVersion("15");
 
-    public Java16TreeDumpTest() {
-        super(new RelevantAttributePrinter(), ".java");
```

Comment:
Note that this is why the reference files have changed that much. The printer of the new superclass filters out deprecated attributes, among other things.